lib/truffle/truffle/cext.rb

```diff
@@ -2347,6 +2347,10 @@ def rb_eval_cmd_kw(cmd, args, kw_splat)
     end
   end
 
+  def rb_error_frozen_object(object)
```

Comment:
Could you add a C API spec in exception_spec.c/rb? (to ensure it works as expected and it's not untested code)

Reply:
Done in ce849ecd46a75bff2d61f31b358f2bb8cce89d85.